langs/node.go

```diff
@@ -51,7 +51,7 @@ const packageJsonContent = `{
 	"author": "",
 	"license": "Apache-2.0",
 	"dependencies": {
-		"@fnproject/fdk": "0.x"
+		"@fnproject/fdk": ">=0.0.11"
```

Comment:
also randomly I saw a build pick up a pre-http-stream version  might just have been docker cache but added this to be sure.